spec.md

```diff
@@ -564,9 +564,25 @@ the resource it creates.
 | service_id* | string | ID of the service from the catalog. MUST be a non-empty string. |
 | plan_id* | string | ID of the plan from the catalog. MUST be a non-empty string. |
 | app_guid | string | Deprecated in favor of `bind_resource.app_guid`. GUID of an application associated with the binding to be created. If present, MUST be a non-empty string. |
-| bind_resource | JSON object | A JSON object that contains data for platform resources associated with the binding to be created. Current valid values include `app_guid` for [credentials](#types-of-binding) and `route` for [route services](#route-services). |
+| bind_resource | JSON object | A JSON object that contains data for platform resources associated with the binding to be created. See [Bind Resource Object](#bind-resource-object) for more information. |
 | parameters | JSON object | Configuration options for the service binding. An opaque object, controller treats this as a blob. Brokers SHOULD ensure that the client has provided valid configuration parameters and values for the operation. |
 
+##### Bind Resource Object
+
+The `bind_resource` object contains platform specific information
```

Comment:
I think it depends on which view of the file is for what user. For example, I think the generated html (e.g. CF docs) and markdown (via github/browser) are for end users. And I agree those should not have 80 col wrapping. Which they don't.  However, I don't expect an end user to ever look at these files in their 'raw' format except when they go into a more of an "expert mode". Kind of like when a user chooses to look at the source (html) of a web page. Most never will. This means that the raw version should be in the form that's best for the user of that form - meaning us.  And long lines, IMO, make it harder on us to do our job since it causes more conflicts and git's diff facility doesn't handle it very well.